addons/mail/static/src/components/composer/composer.js

```diff
@@ -48,11 +48,26 @@ class Composer extends Component {
          * Reference of the text input component.
          */
         this._textInputRef = useRef('textInput');
+        /**
+         * Reference of the subject input. Useful to set content.
+         */
+        // FIXME this ref should not be needed if input bindings is handled in a new component
+        // (task-2284290)
+        this._subjectRef = useRef('subject');
         this._onClickCaptureGlobal = this._onClickCaptureGlobal.bind(this);
     }
 
     mounted() {
         document.addEventListener('click', this._onClickCaptureGlobal, true);
+        // FIXME this whould not be needed if input bindings is handled in a new component
+        // (task-2284290)
+        this._update();
+    }
+
+    patched() {
+        // FIXME this whould not be needed if input bindings is handled in a new component
+        // (task-2284290)
+        this._update();
```

Comment:
I think duplicate comment is a bit overkill on those hooks, just add it once when doing the 
`this._subjectRef.el.value = this.composer.subjectContent;`